functions-and-operators/string-functions.md

```diff
@@ -147,11 +147,107 @@ Return the index position of the first argument within the second argument.
 
 ### [`FORMAT()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_format)
 
-Return a number formatted to specified number of decimal places.
+The `FORMAT(X,D[,locale])` function is used to format the number X rounded to a format similar to "#,###,###. ##", retaining D decimal places, and returning the result as a string.
+
+The function rounds and formats based on the specified number of decimal places. If the specified number of decimal places is greater than the actual number of decimal places in the number, the result is padded with zeros to the corresponding length.
+
```

Comment:
```suggestion
```

Reply:
moved it to https://github.com/pingcap/docs/pull/16085/files#r1472347044 as it is more relevant there